db/fixtures/ae_datastore/ManageIQ/__domain__.yaml

```diff
@@ -6,6 +6,7 @@ object:
     name: ManageIQ
     description: Base domain
     display_name: 
-    system: true
+    system: false
     priority: 0
     enabled: true
+    tenant_id: 1
```

Comment:
@dkorn 
Please delete this file from your pull request, this file should be excluded since it changes the key attributes system => false